packages/vuetify/src/components/VPagination/VPagination.ts

```diff
@@ -105,12 +105,11 @@ export default mixins(Colorable, Themeable).extend({
   },
 
   methods: {
-    init () {
+    async init () {
```

Comment:
we don't use async/await

Reply:
Ok. I'm pretty new to working on this and open source in general. Is there a reason why, just so I can keep it in mind for the future?